packages/desktop/features/analytics.features.ts

```diff
@@ -14,6 +14,9 @@ const analyticsFeatures: IAnalyticsFeatures = {
     dashboardRoute: {
         enabled: true,
     },
+    migration: {
```

Comment:
lets better rename this to `strongholdMigration` because there can be other migrations in the app 🙏🏼